app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    cache_result = dict()
+
+    def inner(*args) -> Any:
+        if args in cache_result:
+            print("Getting from cache")
+            return cache_result[args]
```

Comment:
you must use DRY, and here you can avoid duplicate code like `return cache_result[args]` and `return result`